js/controller.js

```diff
@@ -1598,28 +1602,32 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
 
       //find html5 video
-      if (element.tagName && element.tagName.toLowerCase().indexOf(CONSTANTS.MEDIA_TYPE.VIDEO) != -1) {
```

Comment:
At the beginning of this function (outside of what the diff show by default) we access element[0] without checking if element is null.  Need an extra null check there.